forge/tests/it/config.rs

```diff
@@ -151,6 +154,7 @@ pub fn runner_with_config(mut config: Config) -> MultiContractRunner {
     base_runner()
         .with_cheats_config(CheatsConfig::new(&config, &EVM_OPTS))
         .sender(config.sender)
+        .evm_spec(evm_spec(&config.evm_version))
```

Comment:
this one was fun: the test runner was actually not respecting the `evm_version` set on the `foundry.toml`. Should be the case now